build.gradle

```diff
@@ -155,16 +188,6 @@ install {
 eclipse {
     ext.downloadSources = true
     ext.downloadJavadoc = true
-    ext.sourceCompatibility = JavaVersion.VERSION_1_8
-}
 
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = "1.7"
-    }
+    ext.sourceCompatibility = JavaVersion.VERSION_11

```

Comment:
Not sure this is safe.

Reply:
It'll very likely break things for older devices. Let's revert.